README.md

```diff
@@ -177,6 +177,16 @@ Press **&#8963;&#8984;&#8679;** to display a generated interface, showing proper
 
 Source: [Paul Hudson](https://www.hackingwithswift.com/articles/229/24-quick-xcode-tips)
 
+### Trigger Custom Behaviors
+
+When working in the Interface Builder, you may prefer to have the attributes inspector open and the project navigator closed. But when working on a code file, you might prefer the navigation panels open and the inspectors closed. If you regularly switch between them, you can lose a lot of time using your mouse to make those adjustments, and using normal keyboard shortcuts is tiresome when you have to invoke several for each switch (and some, like showing/hiding the SwiftUI Preview Canvas, have no keyboard shorcut and you can't assign one). Prior to Xcode 12 the state of these panels were independent on each tab. Now, you have to use separate windows if you do not want to open and close the panels constantly.
+
+Xcode Behaviors seems like a good solution to this isssue, as it can set the state of a number of Xcode interface elements when certain events are triggered, like when a build starts or you do a search. Alas, there are no triggers for when you focus on a specific type of file. But you can create custom triggers and assign keyboard shortcuts to them.
+
```

Comment:
This is a great tip, however, personally I find the first two paragraphs quite verbose and I think you might be able to sum them both up in 1 or 2 sentences. Also, I think the instructions paragraph may be easier to read as a bullet point list of steps.

For example:

When working in different views you may want different panels open, which can be time consuming when using your mouse. Xcode behaviours allow you to set the state of your panels using keyboard shortcuts instead.

* Go to `Xcode -> Preferences`
* Open the `Behaviors` tab
* Click `+` at the bottom of the list and name it
* Set the state of the panels on the right such as `Navigator`, `Inspectors`, `Preview` etc.
* Assign a keyboard shortcut next to each name

Now you can set up the panels to your liking depending on what you are doing in Xcode.

Reply:
+1 

I think we should keep all tips brief and concise. 😊

Reply:
Thanks for the feedback. I've shortened the description and put the instructions into a bulleted list. 

However, I did not go bare-bones on the description because my perception is that Behaviors is a relatively unknown and under-used Xcode feature, and a bit of education on what it is might be warranted. And given that, some explanation on why this tip is useful also seems appropriate.

BTW, "keep all tips brief and concise" seems like a great addition to the contribution guidelines. I know it already says "concise", but that seems related to the use of keyboard symbols instead of their names i.e. a preference for shorter nomenclature (as opposed to using fewer words overall). Maybe something along the lines of "Please keep all tips brief and concise, avoiding lengthy explanations or descriptions" as a separate line item would highlight that style preference.

Reply:
Thank you, it looks good and happy with your point that a little more than my incredibly brief example.

I did a very small typo fix and grammar tweak

Reply:
I will add the point to the contributing guide.